developer-extension/src/panel/components/tabs/eventsTab.tsx

```diff
@@ -43,7 +42,13 @@ export function EventTab({ events, filters, onFiltered, clear }: EventTabProps)
           >
             <Chip value="rum">RUM</Chip>
             <Chip value="logs">Logs</Chip>
+            <Chip value="telemetry">Telemetry</Chip>
           </Chip.Group>
+          <Checkbox
+            label="Show only the latest View event"
+            checked={!filters.outdatedVersions}
+            onChange={(e) => onFiltered({ ...filters, outdatedVersions: !e.target.checked })}
+          />
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it seems to be the case with existing checkboxes but could we have a pointer cursor when hovering them (like the other inputs)?

Reply:
Done! (luckily `mantine` supports it 🎉 )